cmd/build.go

```diff
@@ -55,46 +48,52 @@ func Build(ctx context.Context) *cobra.Command {
 			if err := utils.CheckIfDirectory(path); err != nil {
 				return fmt.Errorf("invalid build context: %s", err.Error())
 			}
+			options.Path = path
 
-			if file == "" {
-				file = filepath.Join(path, "Dockerfile")
+			if options.File == "" {
+				options.File = filepath.Join(path, "Dockerfile")
 			}
 
-			if err := utils.CheckIfRegularFile(file); err != nil {
+			if err := utils.CheckIfRegularFile(options.File); err != nil {
 				return fmt.Errorf("invalid Dockerfile: %s", err.Error())
 			}
 
 			buildKitHost, isOktetoCluster, err := build.GetBuildKitHost()
 			if err != nil {
 				return err
 			}
-			log.Information("Running your build in %s...", buildKitHost)
+			if buildKitHost == "" || options.BuildLocally {
+				log.Information("Running your build in docker...")
+			} else {
+				log.Information("Running your build in %s...", buildKitHost)
+			}
 
 			ctx := context.Background()
-			if err := build.Run(ctx, "", buildKitHost, isOktetoCluster, path, file, tag, target, noCache, cacheFrom, buildArgs, secrets, progress); err != nil {
+			if err := build.Run(ctx, "", buildKitHost, isOktetoCluster, options); err != nil {
 				analytics.TrackBuild(buildKitHost, false)
 				return err
 			}
 
-			if tag == "" {
+			if options.Tag == "" {
 				log.Success("Build succeeded")
 				log.Information("Your image won't be pushed. To push your image specify the flag '-t'.")
 			} else {
-				log.Success(fmt.Sprintf("Image '%s' successfully pushed", tag))
+				log.Success(fmt.Sprintf("Image '%s' successfully pushed", options.Tag))
```

Comment:
will the tag show the entire thing? (e.g. okteto.dev/foo/bar:123)

Reply:
It will show the tag that you passed as a flag, that makes the user know that what he has introduced as a flag is actually pushed